### DISCUSSION TITLE
German Localization & Translation

### DISCUSSION THREAD

**Discussion by j13055** [TriliumNext/Trilium#4541] [General]
Is there anone who had translated it to german language?

**Comment by Nriver** [comment]
You can take https://github.com/Nriver/trilium-translation as a start :)

**Reply by j13055** [reply]
thanks

**Reply by j13055** [reply]
Hey, first of all thanks for the work you have done. 
i have translated everything you translated into german. I still got a small problem. There are some parts which are not translated in the released file.
For Example "Owned Attributes" and all the other ribbon buttons. In the translated source code in "trilium-src" it is translated into german, but when i make release it is not. i only tested the windows Desktop version.

**Reply by Nriver** [reply]
Try to clear the cache folder `C:\Users\username\AppData\Roaming\Trilium Notes`. Restart trilium, see if it's updated.

**Reply by j13055** [reply]
Thanks, that worked. 👍
